packages/grid/src/vaadin-grid-sort-mixin.js

```diff
@@ -42,6 +42,19 @@ export const SortMixin = (superClass) =>
           value: () => defaultMultiSortPriority,
         },
 
+        /**
+         * When `true`, Shift-clicking an unsorted column's sorter adds it to the multi-sort.
+         * Shift + Space does the same action via keyboard. This property has precedence over the
+         * `multiSort` property. If `multiSortOnShiftClick` is true, the multiSort property is effectively ignored.
+         *
+         * @attr {boolean} multi-sort-on-shift-click
+         * @type {boolean}
+         */
+        multiSortOnShiftClick: {
```

Comment:
Should also be added to the `d.ts` file and tested in `grid.types.ts`

Reply:
Done.